### ISSUE TITLE
Wrong trainig process rerunning

### ISSUE THREAD

**Issue by BerkovSmartMerch** [ultralytics/yolov5#4198]
## ❔Question
I try to continue the interrupted training, but instead of the desired project, another is launched. How to fix it?

## Additional context
Launched training from the terminal for 300 epochs. I had to interrupt it at epoch 196 and start another training (generally a different project) in the same terminal window, which I also interrupted in the middle. To restore the first training, I write the entire command with which I ran it and add --resume. But the second interrupted learning process starts, not the first. This is understandable both by the number of the epoch from which the project starts, and by the name.

**Comment by glenn-jocher** [comment]
@BerkovSmartMerch 👋 Hello! Thanks for asking about **resuming training**. YOLOv5 🚀 Learning Rate (LR) [schedulers](https://github.com/ultralytics/yolov5/blob/9f3a388cea66ca71b9bbe057929eed8fbc920a7e/train.py#L109-L142) follow predefined LR curves for the fixed number of `--epochs` defined at training start (`default=300`), and are designed to fall to a minimum LR on the final epoch for best training results. For this reason you can not modify the number of epochs once training has started.

<img width="850" alt="LR Curves" src="https://user-images.githubusercontent.com/26833433/119332195-8e113a80-bc88-11eb-8eb7-7494ff1d5c01.png">

If your training was **interrupted** for any reason you may continue where you left off using the `--resume` argument. If your training fully completed, you can start a **new** training from any model using the `--weights` argument. Examples:

## Resume Single-GPU

You may not change settings when resuming, and **no additional arguments** other than `--resume` should be passed, with an optional path to the checkpoint you'd like to resume from. If no checkpoint is passed the most recently updated `last.pt` in your `yolov5/` directory is automatically found and used:
```bash
python train.py --resume  # automatically find latest checkpoint (searches yolov5/ directory)
python train.py --resume path/to/last.pt  # specify resume checkpoint
```

## Resume Multi-GPU

[Multi-GPU DDP trainings](https://docs.ultralytics.com/yolov5/tutorials/multi_gpu_training) must be resumed with the same GPUs and DDP command, i.e. assuming 8 GPUs:
```bash
python -m torch.distributed.run --nproc_per_node 8 train.py --resume  # resume latest checkpoint
python -m torch.distributed.run --nproc_per_node 8 train.py --resume path/to/last.pt  # specify resume checkpoint
```

## Start from Pretrained

If you would like to **start training** from a fully trained model, use the `--weights` argument, not the `--resume` argument:
```bash
python train.py --weights path/to/best.pt  # start from pretrained model
```

Good luck and let us know if you have any other questions!

**Comment by BerkovSmartMerch** [comment]
But I don't change the number of epochs - I just copy the original command with all parameters and add --resume. I have been training this particular model for a long time, and I have previously also restarted it after interrupting training. Now the only difference is that in this window, after the interruption, another learning process was started. And this second process is now started from the place where IT was interrupted, despite the fact that all parameters correspond to the original project. 

Do I understand correctly that the last training process is always resumed - regardless of the parameters specified after "python train.py"? And should I now only start new training with weights after 197 epochs?

**Comment by glenn-jocher** [comment]
@BerkovSmartMerch resuming is very simple. All instructions are in my above post, suggest you follow them.

**Comment by BerkovSmartMerch** [comment]
I read it inattentively, sorry:)
